tls-attestation/collaterals/integration-passport.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
+skinparam sequenceMessageAlign center
+
+participant "TLS A" as tls_a order 10
+participant "Veraison XYZ" as verifier order 20
+participant "TLS B" as tls_b order 30
+participant "PARSEC" as parsec order 50
+participant "KAS" as kas order 50
+
+group KAT discovery
+tls_b --> parsec: get_kat_media_type()
+tls_b <-- parsec: kat-media-type
+end
+
+group Passport issuance
```

Comment:
Sticking with the "TLS B participates in the issuance, and stores the passport" approach? (Not criticising, just curious if TLS B essentially means client app + TLS stack essentially)